src/visual.ts

```diff
@@ -392,55 +392,89 @@ module powerbi.extensibility.visual {
             return result;
         }
 
-        private static getBrokenWords(
+        private static processText(
             words: WordCloudText[],
             stopWords: string[],
             settings: WordCloudSettings): WordCloudText[] {
-
-            const brokenStrings: WordCloudText[] = [],
+            let processedText: WordCloudText[] = [],
+                partOfProcessedText: WordCloudText[] = [],
                 whiteSpaceRegExp: RegExp = /\s/,
                 punctuationRegExp: RegExp = new RegExp(`[${WordCloud.Punctuation.join("\\")}]`, "gim");
 
-            if (!settings.general.isBrokenText) {
-                return words;
-            }
-
             words.forEach((item: WordCloudText) => {
                 if (typeof item.text === "string") {
                     let splittedWords: string[] = item.text
-                        .replace(punctuationRegExp, " ")
+                        .replace(punctuationRegExp, "")
```

Comment:
Why do you remove all punctuation using ```punctuationRegExp``` as a pattern?
As far as I understand, this construction transforms ```Power,BI``` to ```PowerBI```. However, I suppose that this behavior is incorrect.